tests/utils.py

```diff
@@ -0,0 +1,45 @@
+import sys
+from argparse import ArgumentParser
+from obiba_mica import MicaClient
+
+class Utils:
+  # SERVER = 'https://mica-demo.obiba.org'
```

Comment:
Enable mica-demo by default so that Travis build can refer to something that exists.